frames/src/main/java/com/checkout/frames/component/cvv/CvvViewModel.kt

```diff
@@ -68,20 +68,25 @@ internal class CvvViewModel @Inject constructor(
         componentState.cvv.value = this
         paymentStateManager.cvv.update { this }
         componentState.hideError()
+        if (componentState.cvv.value.length == paymentStateManager.cardScheme.value.cvvLength.min()) {
```

Comment:
Will this work for `MAESTRO` scheme? where the minimum CVV length in the set is `0`